src/theme-preval.js

```diff
@@ -258,33 +258,6 @@ const buttons = {
   }
 }
 
-const flash = {
```

Comment:
It's technically a breaking change to remove these values from the theme. Let's hold off on doing that until the "Remove old color variables from theme" step of this [milestone](https://github.com/github/design-systems/issues/1219) so we can batch all the breaking changes together.